src/components/Upload/Uploader.tsx

```diff
@@ -214,8 +214,45 @@ export default function Uploader(props: Props) {
             addLogLine(`received file upload request`);
             setWebFiles(props.webFileSelectorFiles);
         } else if (props.dragAndDropFiles?.length > 0) {
-            addLogLine(`received drag and drop upload request`);
-            setWebFiles(props.dragAndDropFiles);
+            if (isElectron()) {
+                const main = async () => {
+                    try {
+                        addLogLine(`uploading dropped files from desktop app`);
+                        // check and parse dropped files which are zip files
+
+                        const electronFiles = [
+                            ...(await Promise.all(
+                                props.dragAndDropFiles.map(async (file) => {
+                                    addLogLine(`dropped file - ${file.name}`);
```

Comment:
Do we really need to log this information here?

Reply:
No, we can drop it. will remove it